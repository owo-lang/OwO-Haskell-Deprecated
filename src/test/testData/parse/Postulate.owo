module A where
  postulate
    OMG_IN_A : Type
  module B where
    postulate
      OMG_IN_B  : Type
      OMG_IN_B' : Type
    OMG_IN_B' : Type
  postulate
    OMG_IN_A  : Type
    OMG_IN_A' : Type
  OMG_IN_A_ : Type
