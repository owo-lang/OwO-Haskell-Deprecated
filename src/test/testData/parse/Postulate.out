File type: CodeFileType
Top module name: .A
 postulate OMG_IN_A (31 3 5) (39 3 13)
  no pragmas
 submodule .B
  postulate OMG_IN_B (84 6 7) (92 6 15)
   no pragmas
  postulate OMG_IN_B' (107 7 7) (116 7 16)
   no pragmas
  type signature OMG_IN_B' (128 8 5) (137 8 14)
   no pragmas
 postulate OMG_IN_A (161 10 5) (169 10 13)
  no pragmas
 postulate OMG_IN_A' (182 11 5) (191 11 14)
  no pragmas
 type signature OMG_IN_A_ (201 12 3) (210 12 12)
  no pragmas
